### DISCUSSION TITLE
Add more currency

### DISCUSSION THREAD

**Discussion by Simonk007** [inventree/InvenTree#2472] [General]
Hi again,
is it possible to create more currency to the system

**Comment by SchrodingersGat** [comment]
It is! Edit the "config.yaml" configuration file and add the currencies you wish to support:

![image](https://user-images.githubusercontent.com/10080325/146841340-0dfca91a-46db-42a7-a0e2-758a0fb4a3d9.png)

**Reply by Simonk007** [reply]
thx im on it

**Reply by eeintech** [reply]
@Simonk007 Please mark it as answer when you have to chance :smiley:
